### PR TITLE
Update README to show users how to apply bar theme

### PR DESCRIPTION
**Pull request by greed-d**
Even though themes are availabe and are awesome, I was pretty confused on how to apply bar theme,  and I guess it's the same with new users so just putting a little snippet in README about bar theme

### CODE REVIEW

**Comment by Jas-SinghFSU** [comment]
I've covered it in the wiki :smile: : https://hyprpanel.com/configuration/theming.html#button-styles

File: README.md [hunk 1]
```diff
@@ -76,6 +76,28 @@ Or you can add it to your Hyprland config (hyprland.conf) to auto-start with:
 ```bash
 exec-once = ags
 ```
+### Available Themes:
+Hyprpanel has many inbuilt themes with two types (split and non-split), that can be imported using a json file. 
+
+You can import it via GUI from `Dashboard > Settings > Themeing > General Settings`
```
Comment (Jas-SinghFSU) [inline README.md:82]:
```suggestion
You can import them via GUI from `Dashboard > Settings > Theming > General Settings`
```

File: README.md [hunk 2]
```diff
@@ -76,6 +76,28 @@ Or you can add it to your Hyprland config (hyprland.conf) to auto-start with:
...
+
+From command like, to import non-split type you can do:
```
Comment (Jas-SinghFSU) [inline README.md:84]:
```suggestion
From command line, to import non-split type you can do:
```

File: README.md [hunk 3]
```diff
@@ -76,6 +76,28 @@ Or you can add it to your Hyprland config (hyprland.conf) to auto-start with:
...
+![Non-split-gruvbox](./assets/gruv.png)
+
+To import split type you can do:
+```
+ags -r "useTheme('$HOME/.config/ags/themes/<theme_name>_split.json')"
+```
+which looks like : 
+![Non-split-gruvbox](./assets/split_panel.png)
+
+You can see the list of available pre-built theme in `$HOME/.config/ags/themes/`. The theme above is gruvbox
```
Comment (Jas-SinghFSU) [inline README.md:98]:
```suggestion
You can see the list of available pre-built theme in `$HOME/.config/ags/themes/`. The theme in the image is gruvbox and gruvbox_split.
```

File: README.md [hunk 4]
```diff
@@ -76,6 +76,28 @@ Or you can add it to your Hyprland config (hyprland.conf) to auto-start with:
...
+
+
```
Comment (Jas-SinghFSU) [inline README.md:100]:
```suggestion
```

**Comment by Jas-SinghFSU** [comment]
And let's move it out of the instructions section and into the configuration section.

**Comment by greed-d** [comment]
OK

**Comment by greed-d** [comment]
Closing this, as it seems redundant

**Pull request closed without merging**